### ISSUE TITLE
Recursive report does not work

### ISSUE THREAD

**Issue by mpetuska** [mhassan1/yarn-plugin-licenses#14]
When trying to list licenses recursively, the following error is thrown.

```
Type Error: n.map is not a function
        at /.yarn/plugins/@yarnpkg/plugin-licenses.cjs:6:4355
```

Environment:
* Yarn: 3.1.1
* Node: 14
* Linker: node-linker
* Global Cache: true
* OS: MacOS

**Comment by mpetuska** [comment]
I can't unfortunately as it's an enterprise project. But here are some key points of our setup:
* We're using yarn workspaces heavily (nested to 3 layers)
* We're hooking up many workspaces as dependencies to other workspaces.

**Comment by mhassan1** [comment]
I've just pushed up a branch that I believe will fix the issue. Can you run `yarn plugin import https://raw.githubusercontent.com/mhassan1/yarn-plugin-licenses/consider-string-licenses/bundles/@yarnpkg/plugin-licenses.js` and try again?

**Comment by mpetuska** [comment]
I can confirm that it works, thanks. Can you hold off closing this issue until the fix is merged into master, please?

**Comment by mhassan1** [comment]
Merged and released as `v0.7.2`
